fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/operations/CreateNonRdfSourceOperation.java

```diff
@@ -71,4 +78,9 @@ public String getParentId() {
     public void setParentId(final String parentId) {
         this.parentId = parentId;
     }
+
+    @Override
+    public ResourceOperationType getType() {
+        return CREATE;
```

Comment:
Why are we overriding this with the same output? Maybe we don't need the `default` implementation [here](https://github.com/fcrepo4/fcrepo4/pull/1597/files#diff-98af73889d05ae57565ffea92694d7b6R44-R46)?

Reply:
tests were failing without it.

Reply:
good point.